packages/create-emotion/types/index.d.ts

```diff
@@ -0,0 +1,85 @@
+// Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
+// TypeScript Version: 2.3
+
+import * as CSS from 'csstype';
+
+export interface MultiDimensionalArray<T> extends Array<T | MultiDimensionalArray<T>> {}
+
+export type CSSBaseObject = CSS.PropertiesFallback<number | string>;
+export type CSSPseudoObject<C> = { [K in CSS.Pseudos]?: CSSObject<C> };
+export interface CSSOthersObject<C> {
+  [propertiesName: string]: Interpolation<C>;
+}
+export interface CSSObject<C> extends CSSBaseObject, CSSPseudoObject<C>, CSSOthersObject<C> {}
+
+export type FunctionInterpolation<C> = (mergedProps: any, context: C) => Interpolation<C> | void;
+
+export interface ArrayInterpolation<C> extends Array<Interpolation<C>> {}
+
+export type Interpolation<C> =
+  | undefined | null | boolean | string | number
+  | CSSObject<C>
+  | ArrayInterpolation<C>
+  | FunctionInterpolation<C>
+  ;
+
+export type FunctionClassNameArg = () => ClassNameArg | void;
```

Comment:
Same as above. We recently deprecated it in #643 so it would be nice to not have it in the typings.

Reply:
OK. I removed them.